internal/controller/constants.go

```diff
@@ -22,21 +26,38 @@ const (
 	OLSAppRedisDeploymentName = "lightspeed-redis-server"
 	// APIKeyMountRoot is the directory hosting the API key file in the container
 	APIKeyMountRoot = "/etc/apikeys" // #nosec G101
+	// CredentialsMountRoot is the directory hosting the credential files in the container
+	CredentialsMountRoot = "/etc/credentials"
+	// OLSAppCertsMountRoot is the directory hosting the cert files in the container
+	OLSAppCertsMountRoot = "/etc/certs"
 	// LLMApiTokenFileName is the name of the file containing the API token to access LLM in the secret referenced by the OLSConfig
 	LLMApiTokenFileName = "apitoken"
+	// OLSPasswordFileName is the name of the file containing password for its infra
```

Comment:
Should we call this RedisPasswordFileName ? password for infra - not clear I  feel.

Reply:
So the idea behind it was to give the passwords file a generic name that each component of OLS can have its secret written in for mounting at runtime. Example Usage [here](https://github.com/vishnuchalla/lightspeed-operator/blob/auth-redis/internal/controller/ols_app_server_assets.go#L78). Updated the comment to make it clear.